blockreassurance.php

```diff
@@ -245,8 +245,8 @@ public function loadAsset()
      */
     private function folderUploadFilesHasGoodRights()
     {
-        return is_writable($this->folder_file_upload)
-            && is_executable($this->folder_file_upload);
+        // do not check is_executable on windows platform (https://www.php.net/manual/en/function.is-executable.php#refsect1-function.is-executable-notes)
+        return is_writable($this->folder_file_upload) && !(preg_match('/^[a-zA-Z]{1}\:{1}\\\\{1}/', $this->folder_file_upload) !== 1) || is_executable($this->folder_file_upload;
```

Comment:
```suggestion
        return is_writable($this->folder_file_upload) && !(preg_match('/^[a-zA-Z]{1}\:{1}\\\\{1}/', $this->folder_file_upload) !== 1) || is_executable($this->folder_file_upload);
```

Reply:
I think you missed a parenthesis

Reply:
Hi @kpodemski, my bad, I did not pull back @boherm changes and tested it.
Pulled back, tested, all OK.